docs/source/notebooks/02_Extract_geographical_watershed_properties.ipynb

```diff
@@ -0,0 +1,767 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.        k: "{} %".format(int((v / total) * 100 * 10 ** precision) / 10 ** precision)
</pre><p>Is this very different from the <code>np.around</code> function, or just python's <code>round</code> ^ </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ouranosinc/raven/pull/424/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>No idea. I took this directly from either the tests or an existing notebook, but I have no idea what this is doing nor if I can replace with a call to "round". I figured that since it's boilerplate and won't be changed by users, might as well leave it as such?</p>

Reply:
Fixed, now use np.round, thanks!